.yarnrc

```diff
@@ -1 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
 
+
+yarn-path ".yarn/releases/yarn-1.18.0.cjs"
```

Comment:
We should add this file to the gitignore. It broke my install and it appears to be what's breaking the automatic build also. Just removing it allows `yarn install` to work as expected.

Reply:
Will do.